deploy/helm/distributed-compute-operator/values.yaml

```diff
@@ -0,0 +1,79 @@
+# Default values for distributed-compute-operator.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# Number of controller instances to run
+replicaCount: 1
+
+# If true, CRD resources will be installed/uninstalled as part of the Helm chart release
+# Uninstalling CRD resources will DELETE all related custom resources.
+installCRDs: true
+
+# Controller manager configuration
+config:
+  # Prometheus metrics port
+  metricsPort: 8080
+  # Health probe port
+  healthProbePort: 8081
+  # Leader election ensures that only one controller instance is active at a time
+  enableLeaderElection: false
+
+  # Logger enconding can be either 'json' or 'console'
+  logEncoder: ""
+  # Logger level can be either 'debug', 'info', 'error', or any integer value >= 0 which corresponds to custom debug levels of increasing verbosity
+  logLevel: ""
+  # Logger level at which stacktraces are printed can either 'info', 'error', or 'panic'
+  logStacktraceLevel: panic
+  # Development mode enables debug logging, console output and stacktraces suitable for troubleshooting
+  logDevelopmentMode: false
+
+image:
+  repository: ghcr.io/dominodatalab/distributed-compute-operator
```

Comment:
ideally, still would like the registry separated from repo even for external releases